changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 5.0.0-preview
+==============
+- **Enhancements includes support for MSALv3 Cache (which is also or will soon be supported by other libraries such as Python and Java, more detail here https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/MSAL.NET-3-released) as well as enabling users to bring their own browser as part of the Auth Code retrieval.
```

Comment:
>own browser [](start = 291, length = 11)

you can reference this issue if you want: [https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1521](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1521) for web browser

Reply:
cool, ill update